docs/DifferentiableTypes.md

```diff
@@ -121,7 +121,7 @@ public protocol Differentiable {
 }
 ```
 
-Mathematically, `Differentiable` represents a [differentiable manifold]: this is a technical term for smooth-surfaced objects like spheres and generalizes types that are compatible with differentiation, like `Float`, `Double`, [`Tensor`][TensorFlow_Tensor], and `SIMD4<Float>`. This definition comes from differential geometry and is quite technical, and not all details are relevant for most use cases.
+Mathematically, `Differentiable` represents a [differentiable manifold]: this is a technical term for smooth-surfaced objects like spheres and generalizes types that are compatible with differentiation, like `Float`, `Double`, [`Tensor`][https://www.tensorflow.org/guide/tensors], and `SIMD4<Float>`. This definition comes from differential geometry and is quite technical, and not all details are relevant for most use cases.
```

Comment:
Please fix the `TensorFlow_Tensor` link at the bottom of the file:

```suggestion
Mathematically, `Differentiable` represents a [differentiable manifold]: this is a technical term for smooth-surfaced objects like spheres and generalizes types that are compatible with differentiation, like `Float`, `Double`, [`Tensor`][TensorFlow_Tensor], and `SIMD4<Float>`. This definition comes from differential geometry and is quite technical, and not all details are relevant for most use cases.

... // At bottom of file:
[TensorFlow_Tensor]: https://www.tensorflow.org/swift/api_docs/Structs/Tensor
```

Please use the link above instead of https://www.tensorflow.org/guide/tensors, which points to the Python documentation for `tf.Tensor`.

Reply:
I am sorry but I am not sure what you mean. Do you mean that I should replace https://www.tensorflow.org/guide/tensors with https://www.tensorflow.org/swift/api_docs/Structs/Tensor in the file?

Reply:
The link at this location should remain as is, while the link defined at the bottom of the file (“TensorFlow_Tensor”) should be changed to the new URL.

Reply:
Yes, please use https://www.tensorflow.org/swift/api_docs/Structs/Tensor as the link.

But please edit the existing [reference-style link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) at the bottom of the page instead of creating an inline link.